src/lib/Libpython/python_extra.mk

```diff
@@ -40,10 +40,11 @@
 #: Additional objects that have direct dependency on python
 #:---------------------------------------------------------------------------
 
-EXTRA_PYTHON_OBJECTS = common_python_utils.o 
+# FIXME: Do we need this file? its not used anywhere
```

Comment:
I do not have any specific knowledge of the history of this file, but if it is not used it should be gotten rid of unless others know differently.

Reply:
Doesn't seem to impact the build when I remove it. It's the only .mk file in the repo. Agreed, get rid of it.